lib/src/xdg_camera_portal.dart

```diff
@@ -0,0 +1,41 @@
+import 'dart:io';
+
+import 'package:dbus/dbus.dart';
+
+import 'xdg_portal_request.dart';
+
+/// Camera portal.
+class XdgCameraPortal {
+  final DBusRemoteObject _object;
+  final String Function() _generateToken;
+
+  XdgCameraPortal(this._object, this._generateToken);
+
+  /// Check to access camera.
```

Comment:
I [improved the documentation upstream](https://github.com/flatpak/xdg-desktop-portal/pull/857). A cleared description is "Request to gain access to the camera".